extensions/podman/packages/extension/scripts/podman-download.ts

```diff
@@ -29,6 +29,28 @@ export enum DiskType {
   Applehv = 'applehv',
 }
 
+type ResponseManifest = {
+  digest: string;
+  annotations: {
+    disktype: string;
+  };
+  platform: {
+    os: string;
+    architecture: string;
+  };
+};
+
+type ResponseLayers = {
+  digest: string;
+  size: number;
+};
+
+type ResponseJSON = {
```

Comment:
Please specify in the name what reponse json this is about, and probably better to name it `type ManifestsResponse`  (with an S) we can assume this is json.

For the others, no need to prefix Response, let's have ManifestLayers and Manifest.

Reply:
And could we have some comments ? like saying where the request come from to clarify what are the types